File: Course23102019/app/src/main/java/com/example/course23102019/MainActivity.java
package com.example.course23102019;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    private EditText EditTextName;
    private EditText EditTextPassword;
    private Button button;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        EditTextName = findViewById(R.id.editText);
        EditTextPassword = findViewById(R.id.editText2);
        button.setOnClickListener();
    }
}

File: Telegramm/app/src/main/java/ru/gushchin/myapplication/MainActivity.kt
package ru.gushchin.myapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import ru.gushchin.myapplication.databinding.ActivityMainBinding
import ru.gushchin.myapplication.ui.fragments.ChatsFragment
import ru.gushchin.myapplication.ui.objects.AppDrawer

class MainActivity : AppCompatActivity() {

    private lateinit var mBinding: ActivityMainBinding
    private lateinit var mAppDrawer: AppDrawer
    private lateinit var mToolbar: androidx.appcompat.widget.Toolbar

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        mBinding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(mBinding.root) // .root - обращение к самому макету
    }

    override fun onStart() {
        super.onStart()
        initFields()
        initFunc()
    }

    private fun initFunc() { // прописываем всю функциольность нашего активити
        setSupportActionBar(mToolbar) //чтобы отобразился тулбар
        mAppDrawer.create()
        //устанавливаем контейнер в мейн активити
        //commit - это подтверждение
        supportFragmentManager.beginTransaction()
            .replace(R.id.dataContainer,
                ChatsFragment()
            ).commit()

    }


    private fun initFields() { // функция, инициализирующая все значения
        mToolbar = mBinding.mainToolBar
        mAppDrawer = AppDrawer(this,mToolbar)

    }
}
